index.html

```diff
@@ -1909,36 +1909,53 @@ <h3>Defining 'string'</h3>
 <p>[[[#char_def]]].</p>
 </div>
 
-<p>Specifications need to be clear about the encoding and processing of textual data. The recommendations in this section are mutually consistent with those in [[DESIGN-PRINCIPLES]]. In general, specifications should only support well-formed Unicode code point strings and should avoid the use of (or access to) the underlying code units or the use of different character encodings.</p>
+<p>Strings are a fundamental component of the protocols and document formats that make up the Web. Because there are multiple ways of working with strings, different terminology has evolved to support the needs of different specifications. Be sure to understand your specification's needs and use the most appropriate and precise terminology.</p>
+
+<p>The best practices found in this section are mutually consistent with those in [[DESIGN-PRINCIPLES]]. The definitions in this section use terms found in the <cite>Internationalization Glossary</cite> [[I18N-GLOSSARY]]. Some of these definitions are taken directly from [[WEBIDL]], [[INFRA]], or the Unicode glossary; in which case the definitions are quoted verbatim and include links to their source. Please refer to instructions in the Internationalization Glossary for how to import and link definitions in your own specification.</p>
+
+<p>[[UNICODE]] forms the basis for text on the Web, so the basic definition of a string is: <em>a well-formed sequence of zero or more <a>Unicode Scalar Values</a></em>. Because the term 'string' is ambiguous, this definition can be more specifically referred to using one of these definitions:
+
+<ul>
+	<li><a>USVString</a> [[WEBIDL]]</li>
+	<li><a>scalar value string</a> [[INFRA]]</li>
+	<li><code>xsd:string</code> [[XMLSCHEMA11-2]]</li>
+</ul></p> 
 
-<aside class="note" id="char_string_char">
-	<p>Specifications should avoid adding or defining support for <a>legacy character encodings</a> unless there is a specific reason to do so. See also <a href="#char_choosing"></a>.</p>
-</aside>
+
+<p>The [[DOM]], JavaScript (including its APIs and document formats), or certain platforms use the <a>UTF-16</a> <a>character encoding</a> to store and process string data. This can expose the underlying details of the UTF-16 encoding, leading to the need to use a different type of string, the <a>DOMString</a> [[WEBIDL]]. For example, [[INFRA]] defines a string this way:</p>
+
+<p style="background-color:white; border: 1px solid brown; margin:0.5em">A string is a sequence of unsigned 16-bit integers, also known as <a>code units</a></p>
```

Comment:
Maybe add a `padding: 0.5em;` to this box?

Reply:
Done.